<launch>

  <node pkg="robocol_rov" name="imu1" type="imu_node.py" args="imu1 link1 link2">
  </node>

  <node pkg="robocol_rov" name="imu2" type="imu_node.py" args="imu2 link2">
  </node>

  <node pkg="robocol_rov" name="imu3" type="imu_node.py" args="imu3 link1">
  </node>

  <node pkg="robocol_rov" name="link1" type="link_node.py" args="link1 imu1 imu3">
  </node>

  <node pkg="robocol_rov" name="link2" type="link_node.py" args="link2 imu1 imu2">
  </node>

</launch>
